PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -292,41 +292,36 @@ extension HomeViewModel {
     }
 
     private func select(recommendation: Recommendation, at indexPath: IndexPath) {
-        tracker.track(
-            event: SnowplowEngagement(type: .general, value: nil),
-            contexts(for: recommendation, at: indexPath)
-        )
-
         let viewModel = RecommendationViewModel(
             recommendation: recommendation,
             source: source,
             tracker: tracker.childTracker(hosting: .articleView.screen),
             pasteboard: UIPasteboard.general
         )
 
-        if let item = recommendation.item, item.shouldOpenInWebView {
-            selectedReadableType = .webViewRecommendation(viewModel)
+        guard let item = recommendation.item else {
```

Comment:
looks like the test is failing here due to having `item == nil`, prob need to build the mock data properly
![image](https://user-images.githubusercontent.com/6743397/221005849-e8e3d3d7-5a92-4ce2-ba65-6c7e800a5a22.png)